### ISSUE TITLE
Fix Vector PERFORMANCE_MODE vs non-PERFORMANCE_MODE

### ISSUE THREAD

**Issue by sushihangover** [PlayScriptRedux/playscript#15]
Vector.cs in _root provides a PERFORMANCE_MODE compile option which needs to be disabled to run the Away3d examples without exception (or modifying the example/core core) but when removing the define other problems with index out of bounds arise due to things like this:

```
#if PERFORMANCE_MODEz && DEBUG
                if (i >= mCount) {
                    throw new IndexOutOfRangeException();
                }
#elif PERFORMANCE_MODEz
#else
                if (i >= mCount) {
                    expand((uint)(i+1));
                }
#endif
```

**Comment by sushihangover** [comment]
commit ca5cd439d8167a1c977f80cf2d16d9e0f28589c4
Author: SushiHangover sushihangover@outlook.com
Date:   Thu Sep 3 03:32:42 2015 -0700

```
Changed PERFORMANCE_MODE in the Vector class to VECTOR_PERF_MODE

This mode is OFF by default now and closes Issue #15

PERFORMANCE_MODE is still exists in Array.cs and flash/utils/Dictionary
```

**Comment by sushihangover** [comment]
Away3d examples now run without changing to a non-PERFORMANCE_MODE pscorlib.dll
